test/BlockChainHelper.cpp

```diff
@@ -354,6 +354,16 @@ void TestBlock::verify(TestBlockChain const& _bc) const
 	{
 		_bc.interface().sealEngine()->verify(CheckNothingNew, m_blockHeader, BlockHeader(), &m_bytes);
 	}
+	catch (Exception const& _e)
+	{
+		u256 daoHardfork = _bc.interface().sealEngine()->chainParams().u256Param("daoHardforkBlock");
+		if (m_blockHeader.number() >= daoHardfork && m_blockHeader.number() <= daoHardfork + 9)
```

Comment:
Such a check also has to go after the try/catch, otherwise we do not check the case where the exception was not thrown but should have been.